tests/integration/scenarios/out_s3/tests/test_out_s3_001.py

```
@@ -218,3 +221,23 @@ def test_out_s3_otlp_json_uploads_signal_payloads(signal_type, json_file, root_k
 
     rendered = json.dumps(payload)
     assert expected_value in rendered
+
+
+def test_out_s3_default_retry_exhausted_action_quarantines_file():
+    store_dir = "/tmp/fluent-bit-test-suite-s3-retry-exhausted"
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/fluent-bit-test-suite-s3-retry-exhausted"

(S108)